lib/core/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -126,6 +127,25 @@ class Malformed t where
 --
 -- Class instances (PathParam)
 --
+instance Wellformed (PathParam (QueryParam "state" (ApiT AddressState))) where
+    wellformed = PathParam "?state=used"
+
+instance Malformed (PathParam (QueryParam "state" (ApiT AddressState))) where
+    malformed = first PathParam <$>
```

Comment:
@KtorZ, I'm not able to make it work for the AddressState, which is a QueryParam in the form of "?state=used" Added this Malformed instance for `AddressState` but no new test cases are being generated in `ApiSpec.hs`. I have also tried:
```
instance Malformed (PathParam (ApiT AddressState))
```
but still no luck.

Reply:
QueryParams are not yet captured by the generic tests, cf: https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/test/unit/Cardano/Wallet/ApiSpec.hs#L343-L350

Reply:
OK, I have wrapped up this PR without `QueryParams`. All `PathParam` and `BodyParam` cases should be moved and covered in `ApiSpec.hs` though.